frontend/src/components/Input/Input.tsx

```diff
@@ -55,14 +56,28 @@ export const Input = forwardRef<InputProps, 'input'>((props, ref) => {
 
   // Return normal input component if not success state.
   if (!props.isSuccess) {
-    return (
-      <ChakraInput
-        ref={ref}
-        {...preventDefault}
-        {...inputProps}
-        sx={props.sx ?? inputStyles.field}
-      />
-    )
+    if (props.prefix) {
+      return (
+        <InputGroup>
+          <InputLeftAddon pointerEvents="none" children={props.prefix} />
+          <ChakraInput
+            ref={ref}
+            {...preventDefault}
+            {...inputProps}
+            sx={props.sx ?? inputStyles.field}
+          />
+        </InputGroup>
```

Comment:
Do we need guarantee that overall width is respected if it is provided for the Input props?

Reply:
What do you mean by this actually? 😅 Do you mean the 6 character width of the OTPs?

Reply:
No, I mean on the layout. As in, if someone uses an Input component at whatever width, and we pass the prop as-is to the ChakraInput within the InputGroup, does having a prefix mean the overall width of the InputGroup be longer than the consumer intends?

Reply:
Hm, do we want the generalised `Input` component to have that level of awareness though? We can have a check on the `OtpForm` component though, if we want to be sure of the prefix width.

Reply:
As discussed, this is not really a concern as `inputProps` is defined to include the props `isSuccess`, `isPrefilled` and `preventDefaultOnEnter`, which does not include width properties which might be affected by the adding of `InputLeftAddon`.